generators/python_functions/html_unescape/README.md

```diff
@@ -0,0 +1 @@
+This module unescapes HTML characters in a text. For example, the text "Here\&#39;s how \&quot;Kern.ai Newsletter\&quot; did today." will be converted to "Here's how "Kern.ai Newsletter" did today."
```

Comment:
If you like, you can also add your name here and say that it was a community contribution! 👍